NetKAN/REKT.netkan

```diff
@@ -0,0 +1,19 @@
+{
+  "spec_version": "v1.4",
```

Comment:
This will need to be changed for the find_regexp

    "spec_version": "v1.10",